lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -351,7 +383,7 @@ module.exports = {
                 "RequestParameters" : {},
                 "Integration" : {
                   "IntegrationHttpMethod" : "POST",
-                  "Type" : "AWS",
+                  "Type" : "${integrationType}",
```

Comment:
To tell you the truth I'm not a big fan of this approach though when working with objects. It's hard to debug if you're missing a comma or something (because it's treated as JSON), plus we're gonna parse anyway to merge it with the entire CF template. I hope we can switch all our templates to just plain JS objects unless there's a pressing reason not to